docs/schema/builtin-element-resultitemtype-complextypesps15xsdsearchset2.md

```diff
@@ -16,12 +16,10 @@ description: "Last modified: March 09, 2015"
   
 ## Element information
 
-|||
-|:-----|:-----|
-|**Element type** <br/> |xs:boolean  <br/> |
-|**Namespace** <br/> |http://schemas.datacontract.org/2004/07/Microsoft.Office.Server.Search.Administration  <br/> |
-|**Schema file** <br/> |schema_Microsoft.Office.Server.Search.Administration.xsd  <br/> |
-   
+| Element Type | Namespace | Schema File |
```

Comment:
This is inconsistent with all other files in this PR... don't transpose rows=>columns. Default to what's already there in the docs.